lib/dest/writeContents/index.js

```diff
@@ -60,17 +60,16 @@ function writeContents(writePath, file, cb) {
   }
 
   function isErrorFatal(err) {
+    var isFatal = true;
```

Comment:
yeah, the rule is "no else after return" i think - might be a different way to do this